.circleci/config.yml

```diff
@@ -54,6 +54,8 @@ jobs:
             # fallback to using the latest cache if no exact match is found
               - v2-dependencies-
         - run: printf "scm.url=scm:git:https://github.com/goshippo/shippo-java-client.git\nscm.tag=release" > release.properties
+        # load the GPG key again as this is a new shell
+        - run: printf -- "$gpg_secret" | gpg --batch --yes --passphrase=$gpg_password --pinentry-mode loopback --import
```

Comment:
this was the root of the issue with the release job.  The key was imported in the build step, but not in the deploy step, which runs in a different shell; so the deploy step was defaulting to some old key from the VM image, which has expired.